blog/admin.py

```diff
@@ -1,3 +1,4 @@
 from django.contrib import admin
+from .models import Post
```

Comment:
it can be a trouble in the feature with relative import. It's better to use absolute import